modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/TranslationManager.testcase

```diff
@@ -40,6 +40,44 @@ definition {
 		}
 	}
 
+	@description = "LPS-148143. Add language with no success message and undo option."
+	@priority = "3"
+	test AddLanguageWithNoSuccessMessageAndUndoOption {
```

Comment:
I think it's better to update the test name to something like `AssertNoSuccessMessageAndUndoOptionAfterAddingLanguage`, because the success message and undo button shouldn't display at all after we adding any language, but the name now looks like it is adding a language that doesn't have success message and undo button, and give me an illusion that  there are languages that have success message and undo button.

Reply:
Thanks, I edit it.